api/management/README.md

```diff
@@ -0,0 +1,200 @@
+claims under the `policy-added-claims` EAR extension.
+
+A policy is defined by a text document describing the rules to be applied by
+the policy engine. Currently, Veraison only support the OPA policy engine.
+Correspondingly, policies must be written in Rego language used by OPA. See
+[OPA policy README] in services repo for details.
+
+Policies are automatically retrieved, based on the attestation scheme and
+tenant id for the evidence being evaluated. Currently, only one policy is
+allowed per tenant/scheme combination. If a policy is added for a tenant/scheme
+combination with an existing policy, the new policy replaces the old one. This
+also increments the internal policy version associated with the tenant/scheme
+combination.
+
+The policy management API exposes each scheme's policy via a unique resource
+path in the form `/policy/v1/{scheme}`. It supports the basic CRUD operations,
+typical to any resource management, in the following way:
+
+- **create**: A new policy is added to a scheme via a POST request to its path.
+- **read**: Policy can be retrieved via a GET request to its path.
+- **update**: A direct update is not possible, however a new policy can be
+              added and activated. Activating the new policy will deactivate
+              the old one, effectively replacing it.
+- **delete**: True deletion is not possible, in order to allow retrieval of the
+              specific policy version that was used to generate an attestation
+              result at a later date. However, all policies may be deactivated,
+              resulting in no policy being used during verification.
+
+[Attestation Results]: https://www.rfc-editor.org/rfc/internet-drafts/draft-fv-rats-ear-00.html
+[OPA Policy README]: https://github.com/veraison/services/blob/main/policy/README.opa.md#rules
+
+## REST API
+
+### Adding a new policy
+
+- A policy is added for a new scheme (with tenant implicit via auth).
+- Optionally, a name for the policy may be specified via query arguments. If a
+  name is not specified, the default name of `"default"` will be used.
+- On success, the new policy object is returned, with Location header set to
+  its unique URI.
+- The newly added policy is inactive. It must be activated via  a separate
+  request (see below).
+
+#### Request:
+```http
+POST /management/v1/policy/PSA_IOT&name=base HTTP/1.1
+Host: veraison.example
+Content-Type: application/vnd.veraison.policy.opa
+Accept: application/vnd.veraison.policy+json
+
+hardware = "AFFIRMING" {
+    // secured state
+    state := bits.rsh(bits.and(evidence["psa-security-lifecycle"], 0xFFFF0000), 8)
+    state >= 0x3000
+    state <= 0x30FF
+} else = "CONTRAINDICATED"
+```
+
+#### Response:
+```http
+HTTP/1.1 201 Created
+Content-Type: application/vnd.veraison.policy+json
+Location: /management/v1/policy/PSA_IOT/340d22f7-9eda-499f-9aa2-5af295d6d812
+
+{
+  "type": "opa",
+  "name": "base",
+  "uuid": "340d22f7-9eda-499f-9aa2-5af295d6d812",
```

Comment:
It makes sense to call this either just "id" or `policy_id` from interface perspective.
Format UUID is fine as type

Reply:
There are already an umber of "policy id"'s floating around. This is less ambiguous.